lib/guardian/plug.ex

```diff
@@ -117,6 +117,7 @@ defmodule Guardian.Plug do
     case Guardian.encode_and_sign(object, type, new_claims) do
       {:ok, jwt, full_claims} ->
         conn
+        |> Plug.Conn.configure_session(renew: true)
```

Comment:
I'm not sure if it makes a difference...but maybe put this after the call to `put_session` ?